[debug] [2023-11-10T14:04:02.554Z] ----------------------------------------------------------------------
[debug] [2023-11-10T14:04:02.564Z] Command:       /home/usuario/.nvm/versions/node/v18.16.1/bin/node /home/usuario/.nvm/versions/node/v18.16.1/bin/firebase login
[debug] [2023-11-10T14:04:02.567Z] CLI Version:   12.5.2
[debug] [2023-11-10T14:04:02.568Z] Platform:      linux
[debug] [2023-11-10T14:04:02.569Z] Node Version:  v18.16.1
[debug] [2023-11-10T14:04:02.571Z] Time:          Fri Nov 10 2023 11:04:02 GMT-0300 (hora estÃ¡ndar de Argentina)
[debug] [2023-11-10T14:04:02.571Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-11-10T14:04:02.589Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] Already logged in as deborahmosqueda@gmail.com
[debug] [2023-11-10T14:04:03.082Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2023-11-10T14:04:03.083Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2023-11-10T14:04:12.660Z] Error: Timed out.
    at Timeout._onTimeout (/home/usuario/.nvm/versions/node/v18.16.1/lib/node_modules/firebase-tools/lib/utils.js:254:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
